MyCoreApp/Pages/Index.cshtml

```diff
@@ -6,5 +6,7 @@
 
 <div class="text-center">
     <h1 class="display-4">Hello Dilshan!</h1>
+    <p>This is my application</p>
+    <br />
```

Comment:
P tag alread is a block level element. So we can remove the br tag.

Reply:
I have made the change